scripts/dataController.js

```diff
@@ -1,23 +1,26 @@
 'use strict';
 
-class DataController {
+(function(module){
+  class DataController {
+  }
 
-}
+  DataController.load = loadLocalStorage
+  DataController.save = saveLocalStorage
 
-DataController.load = loadLocalStorage
-DataController.save = saveLocalStorage
+  function loadLocalStorage() {
+    let rawData = localStorage.getItem('masters3d');
+    if (rawData !== null) {
+      let data = JSON.parse(rawData)
+      return {success:true, data}
+    } else {
+      return {success:false, data:{}}
+    }
+  }
 
-function loadLocalStorage() {
-  let rawData = localStorage.getItem('masters3d');
-  if (rawData !== null) {
-    let data = JSON.parse(rawData)
-    return {success:true, data}
-  } else {
-    return {success:false, data:{}}
+  /** @param {Object} data */
+  function saveLocalStorage(data) {
+    localStorage.setItem('masters3d', JSON.stringify(data))
   }
-}
 
-/** @param {Object} data */
-function saveLocalStorage(data) {
-  localStorage.setItem('masters3d', JSON.stringify(data))
-}
\ No newline at end of file
+  module.DataController = DataController
+})(window)
```

Comment:
Perhaps I could expand this to make the whole app be encapsulated into an APP variable.